### PR TITLE
Added a module for converting to/from dicts

### PR DESCRIPTION
**Pull request by jesper-friis**
# Description:
Added a module for converting to/from dicts

## Type of change:

- [ ] Bug fix.
- [x] New feature.
- [ ] Documentation update.
- [ ] Testing.

## Checklist for the reviewer:

This checklist should be used as a help for the reviewer.

- [ ] Is the change limited to one issue?
- [ ] Does this PR close the issue?
- [ ] Is the code easy to read and understand?
- [ ] Do all new feature have an accompanying new test?
- [ ] Has the documentation been updated as necessary?

### CODE REVIEW

**Review by Treesarj — APPROVED** [review]

**Review by Treesarj — COMMENTED** [review]
Looks good but build is failing(pls fix)

File: tripper/convert/convert.py [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+    recognised_keys: "Optional[Union[Dict, str]]" = None,
+    keep: bool = False,
+) -> None:
+    """Save a dict to a triplestore.
+
+    Arguments:
+        dct: The dict to be saved.
+        iri: IRI of indicidual that stands for the dict.
+        bases: Parent class(es) or the dict.
+        lang: Language to use for keys.
```
Comment (CasperWA) [inline tripper/convert/convert.py:136]:
```suggestion
```
Also, add documentation for `ts` parameter.

**Review by CasperWA — CHANGES_REQUESTED** [review]
this might fix the docs build issue?

File: tripper/convert/convert.py (hunk 1 above)
Comment (CasperWA) [reply tripper/convert/convert.py:138]: Conclusion: It needed to be added to the function signature.

**Review by CasperWA — APPROVED** [review]

**Pull request merged**